static/js/services/enketo-translation.js

```diff
@@ -352,8 +343,7 @@ angular.module('inboxServices').service('EnketoTranslation', [
       }
 
       withElements(root.childNodes)
-        // TODO repeat-relevant may be unnecessary if https://github.com/enketo/enketo-core/issues/336 is resolved
-        .filter(without('meta', 'inputs', 'repeat', 'repeat-relevant'))
```

Comment:
Do we want to deprecate this, but continue filtering `repeat-relevant` nodes until we've removed them from all the production forms?

Reply:
We could do, but in my tests it made no difference - the field didn't exist in the created contact or its children. How about I put a line in the release notes about removing the `repeat-relevant` nodes?

Reply:
I'll add a warning to `medic-conf` flagging it as unnecessary with webapp `2.14.0` and later.

Reply:
Nice one.